requirements.txt

```diff
@@ -1,6 +1,8 @@
 falcon==1.1.0
+
+keystoneauth1>=2.21.0  # Apache-2.0
 oslo.config>=3.22.0 # Apache-2.0
-oslo.config>=3.22.0  # Apache-2.0
+oslo.context>=2.14.0 # Apache-2.0
```

Comment:
This is not directly a comment for this PR, but I'd like to separate our indirect dependencies from our direct dependencies so that we can maintain them more easily.  One way we could approach this would be to maintain two requirements files, one with direct dependencies and one with indirect dependencies.  Then we can have a tox task that updates the full set (requirements.txt).  This should give us good control over the libraries we are using.

Also, I think we need to specify all our dependencies exactly via `==` rather than with ranges (again, this is not directly related to this PR).